server/netty-server/src/main/scala/sttp/tapir/server/netty/internal/NettyRequestBody.scala

```diff
@@ -36,7 +39,44 @@ class NettyRequestBody[F[_]](createFile: ServerRequest => F[TapirFile])(implicit
             Files.write(file.toPath, requestContentAsByteArray)
             RawValue(FileRange(file), Seq(FileRange(file)))
           })
-      case _: RawBodyType.MultipartBody => ???
+      case m: RawBodyType.MultipartBody =>
+        new SequenceSupport[F]()
+          .sequence[Part[Any]](getParts(serverRequest, m))
+          .map(RawValue.fromParts(_))
+    }
+  }
+
+  private def getParts(serverRequest: ServerRequest, m: RawBodyType.MultipartBody): List[F[Part[Any]]] = {
```

Comment:
Consider placing `nettyRequest` function before `getParts` to match the order of appearance in `toRaw` and make it easier to read.

Reply:
done